atomic_reactor/plugins/post_koji_import.py

```diff
@@ -15,20 +15,21 @@
 from tempfile import NamedTemporaryFile
 from typing import Any, Dict, Iterator, List, Optional, Tuple
 
+import koji_cli.lib
+
 from atomic_reactor.config import get_koji_session, get_openshift_session
 from atomic_reactor import start_time as atomic_reactor_start_time
 from atomic_reactor.plugin import PostBuildPlugin
 from atomic_reactor.plugins.pre_add_help import AddHelpPlugin
 from atomic_reactor.source import GitSource
-from atomic_reactor.plugins.build_orchestrate_build import (get_worker_build_info,
-                                                            get_koji_upload_dir)
+from atomic_reactor.plugins.build_orchestrate_build import get_worker_build_info
```

Comment:
We should not be importing this anymore. Should be fine to address this in a separate PR.